packages/ffe-form-react/src/BaseRadioButton.js

```diff
@@ -38,7 +41,7 @@ class BaseRadioButton extends Component {
             className,
         );
 
-        const isSelected = checked || selectedValue === value;
+        const isSelected = selectedValue ? selectedValue === value : checked;
```

Comment:
Since `selectedValue` will now override `checked` if both are passed as props instead of the other way around, this should probably be a breaking change. Adding **BREAKING CHANGE** to the commit-message along with a description of what's changed will take care of that. 

It might cause regressions if anyone was relying on the previous behaviour so it's better to be explicit about it.

Reply:
Added "BREAKING CHANGE" to the commit message

Reply:
Then I don't have any further objections :)